ci/gpu/build.sh

```diff
@@ -37,7 +37,7 @@ logger "Activate conda env..."
 source activate gdf
 conda install -c rapidsai/label/cuda${CUDA_REL} -c rapidsai-nightly/label/cuda${CUDA_REL} cudf=${CUDF_VERSION} rmm=${RMM_VERSION} nvstrings=${NVSTRINGS_VERSION}
 conda install -c conda-forge lapack cmake==3.14.3 umap-learn
-conda install -c teju85 libclang
+conda install -c rapidsai libclang
```

Comment:
These conda installs should all be a single command

Reply:
Done @raydouglass